main.go

```diff
@@ -27,9 +32,20 @@ func main() {
 		klog.Fatal(configError)
 	}
 
+	//Check and verify token:
+	config.KubeClient()
+
 	database.GetConnection()
+
+	router := chi.NewRouter()
```

Comment:
The indexer is using the [gorilla/mux router](https://github.com/gorilla/mux). I want to be consistent across our projects.

I haven't evaluated these libraries. It's ok to change if `chi` is better, but we should update our other projects for consistency.

Reply:
Agree we should have consistency. I only chose chi because it seems to be the most updated of the two projects and it looks like mux is looking for [new maintainers](https://github.com/gorilla/mux/issues/659).

Reply:
changed to gorilla mux